web-app/helpers/certificate-designs.ts

```diff
@@ -31,23 +27,36 @@ function addText(canvas: Canvas, text: string, font: string, fillStyle: string,
 export async function populateDeveloperCert(canvas: Canvas, details: any) {
   // TODO
   console.log('populateDeveloperCert', { details });
-  const { tokenId, date, programName, accountName, expiration } = details;
+  const { tokenId, date, programName, accountName, expiration, programDescription, instructor } = details;
 
-  const fillStyle = `#${Math.floor(Math.random() * 16777215).toString(16)}`; // This is just temporary and will be removed once we have final designs for each program's certificate. https://css-tricks.com/snippets/javascript/random-hex-color/
-  const font = `40px '${fontFamily}' bold`;
+  const newDate = date.split(' ')[0];
```

Comment:
In general, I try to avoid hard-coding things like `[0]` and `[1]`. For this particular line, if what you're doing is just trying to change the appearance of a date, search the repo for `dayjs`. Format the date with that kind of helper.

Reply:
done